fhirzeug/generators/python_pydantic/static_files/README.md

```diff
@@ -1 +1,37 @@
 This stub for FHIR generated by [fhirzeug](https://github.com/skalarsystems/fhirzeug).
+
+# Format
+
+All profiles are in one file.
+
+# FHIR Specific JSON Representation
+
+Generally this generated code tries to stick as close as possible to the [FHIR JSON spec](https://www.hl7.org/fhir/json.htm). Another important is the [FHIR Datatypes spec](https://www.hl7.org/fhir/datatypes.html).
+
+## Empty Strings
+
+> String property values can never be empty. Either the property is absent, or it is present with at least one character of content.
+
+Additionally whitespaces are stripped:
+
+> Note: This means that a string that consists only of whitespace could be trimmed to nothing, which would be treated as an invalid element value. Therefore strings SHOULD always contain non-whitespace content. - https://www.hl7.org/fhir/datatypes.html#primitive
+
+That means empty strings are interpreted as `null` values. This could lead to invalid arrays (`[""]`).
```

Comment:
I'd add info why invalid - because we decided (?), that we don't allow `null`s as array items.